app/api/admin_translations.py

```diff
@@ -0,0 +1,19 @@
+from flask import send_file, Blueprint
+import shutil
+import uuid
+import tempfile
+
+admin_blueprint = Blueprint('admin_blueprint', __name__, url_prefix='/admin/content/translations/all')
+temp_dir = tempfile.mkdtemp()
+
+@admin_blueprint.route('/', methods=['GET'])
+def download_translations():
+    """Admin Translations Downloads"""
+    uuid_literal = uuid.uuid4()
+    shutil.make_archive('translations{}'.format(uuid_literal), 'zip', temp_dir)
```

Comment:
Extract the filename in one variable and replace 3 occurrences

Reply:
All of them are not the same thing I believe. There are some differences.

Reply:
@